instrumentation/postgresql-simple/hs-opentelemetry-instrumentation-postgresql-simple.cabal

```diff
@@ -39,6 +39,7 @@ library
     , postgresql-libpq
     , postgresql-simple
     , text
+    , unliftio-core
```

Comment:
@magthe please edit _package.yaml_ instead of this.

Reply:
I'm hoping the change I pushed addresses this.